packages/apps/kadena-transfer/tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
+  "extends": "./node_modules/@kadena-dev/heft-rig/profiles/default/tsconfig-base.json",
+  "compilerOptions": {
+    "baseUrl": "src",
+    "target": "es5",
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+    "noFallthroughCasesInSwitch": true,
+    "module": "esnext",
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "noEmit": true,
+    "jsx": "react-jsx"
+  },
+  "exclude": [
+    "node_modules",
+    "**/*.spec.ts",
+    "**/*.spec.tsx",
+    "**/*.test.ts",
+    "**/*.test.tsx"
+  ],
+  "include": ["src"]
```

Comment:
Are these all different from the `tsconfig-base.json`? If so, why? If not, let's remove the ones that aren't part of it. We probably need something similar to [this one](https://github.com/kadena-community/kadena.js/blob/a60b34845c079dfbcc4ccd0cab63780e13369162/packages/apps/kadena-docs/tsconfig.json)

We can also create a new profile for `nextjs` in `@kadena-dev/heft-rig/profiles/nextjs`. This would benefit other apps as well